python/python/glide/async_commands/sorted_set.py

```diff
@@ -240,3 +380,40 @@ def _create_z_cmd_store_args(
         args.append(aggregation_type.value)
 
     return args
+
+
+def _create_geosearch_args(
+    key: str,
+    search_from: Union[str, GeospatialData],
+    seach_by: Union[GeoSearchByRadius, GeoSearchByBox],
+    order_by: Optional[OrderBy] = None,
+    count: Optional[GeoSearchCount] = None,
+    with_coord: bool = False,
+    with_dist: bool = False,
+    with_hash: bool = False,
+) -> List[str]:
+    args = [key]
+    if isinstance(search_from, str):
+        args += ["FROMMEMBER", search_from]
+    else:
+        args += [
+            "FROMLONLAT",
+            str(search_from.longitude),
+            str(search_from.latitude),
+        ]
+
+    args += str(seach_by).split()
```

Comment:
If you change `__str__` to `to_args`
```suggestion
    args += seach_by.to_args()
```